src/settings-web/src/icons/config/fabric-icons.json

```diff
@@ -5,9 +5,6 @@
   "excludeThirdPartyIcons": false,
   "chunkSubsets": false,
   "hashFontFileName": false,
-  "subsetChunkSettings": {
-    "maxSubsetSize": 100
-  },
```

Comment:
@enricogior @crutkas Was there any reason for having this in the fabric-icons.json file? It doesn't get auto-generated with any of the options available on the fabric icons site and the chunkSubsets was also set to false so I'm not sure where subsetChunkSettings would be used.

Reply:
honestly no clue

Reply:
What the reason to remove it? Is it preventing something to work?

Reply:
I wasn't able to get it in the auto-generated file with any of the configurations that could be set online. I guess I can just retain it in the json file by adding it back manually but I'm unsure if it has any effect.

Reply:
Why do we need all this changes? Can it be done as it was done for the other icons, changing only https://github.com/microsoft/PowerToys/blob/master/src/settings-web/src/setup_icons.tsx ?

Reply:
In this case there was no appropriate icon for ImageResizer within the existing subset, so a new subset needs to be made with the required icons (using [these steps](https://github.com/microsoft/PowerToys/blob/master/doc/devdocs/settings-web.md))

Reply:
I don't understand why, the icons can be loaded directly, afaik they don't need to be added to any subset.
You can simply import the icon and register it as we do for the other modules.
```
import FancyZonesSVG from './svg/fancy_zones.svg';
import PowerRenameSVG from './svg/power_rename.svg';
import ShortcutGuideSVG from './svg/shortcut_guide.svg';

  registerIcons({
    icons: {
      'pt-fancy-zones': ( <FancyZonesSVG /> ),
      'pt-power-rename': ( <PowerRenameSVG /> ),
      'pt-shortcut-guide': ( <ShortcutGuideSVG /> ),
      'pt-power-preview': ( <Icon iconName="FabricReportLibrary" /> ),
    }
  });
```
Btw, also `pt-power-preview` should have done the same.

Reply:
@crutkas @enricogior Is there a way to get the SVG version of the icon from Office UI fabric? I can avoid changing the subset if I'm able to generate that.

Reply:
In what format do you have the icon?

Reply:
I don't have it in a separate file, it's part of the fabric icons subset file which is in .woff format and can be accessed using the Icon syntax in the code snippet above. I can get a separate subset with just the icon I want in a .woff file but I'll have to check if there is a way to change it to svg.

Reply:
check teams :)

Reply:
@enricogior switched to SVG. Now only setup_icons and the dist/bundle.js files have changes in the settings projects. Thanks for calling that change out.

Reply:
Great!